components/frontend/src/dynamo/frontend/main.py

```diff
@@ -262,6 +268,8 @@ async def async_main():
         kwargs["tls_key_path"] = flags.tls_key_path
     if flags.namespace:
         kwargs["namespace"] = flags.namespace
+    if flags.request_template:
+        kwargs["template_file"] = flags.request_template
 
```

Comment:
In case the rest of the PR doesn't land, I would be happy for us to take just this file. It brings the python CLI closer to `dynamo-run` (https://github.com/ai-dynamo/dynamo/blob/main/launch/dynamo-run/src/flags.rs#L139), and that flag is already useful for chat completions.